Sources/Exporters/OpenTelemetryProtocol/trace/OtlpTraceExporter.swift

```diff
@@ -16,14 +17,17 @@ public class OtlpTraceExporter: SpanExporter {
     let config : OtlpConfiguration
     var callOptions : CallOptions? = nil
 
-    public init(channel: GRPCChannel, config: OtlpConfiguration = OtlpConfiguration()) {
+    public init(channel: GRPCChannel, config: OtlpConfiguration = OtlpConfiguration(), logger: Logger = Logger(label: "io.grpc", factory: { _ in SwiftLogNoOpLogHandler() }), envVarHeaders: [(String,String)]? = EnvVarHeaders.attributes) {
```

Comment:
Signature changed to include the logger and envVarHeaders.  Both include default values that match the older behavior. This is a non-breaking change for existing clients.